config/initializers/inflections.rb

```diff
@@ -15,6 +15,7 @@
   inflect.acronym 'GI'
   inflect.acronym 'IHub'
   inflect.acronym 'MDOT'
+  inflect.acronym 'MPI'
```

Comment:
this is an entirely new acronym (instead of changing one that already exists).  There already exist some request specs and controller specs for endpoints in our system, and none of them have been broken.  To the best of my ability, I think the serialized data is unchanged